tests/src/testCommon.hh

```diff
@@ -216,12 +217,24 @@ template <typename NonLinearOperator, typename FiniteElement>
                                 .addResultRequest(ResultType::linearStress);
 
   ResultTypeMap<double> result;
-  auto gridView        = fe.localView().globalBasis().gridView();
+  auto gridView  = fe.localView().globalBasis().gridView();
+  using GridView = decltype(gridView);
+  Dune::VtkWriter<GridView> vtkWriter(gridView);
   auto scalarBasis     = makeConstSharedBasis(gridView, lagrangeDG<1>());
   auto localScalarView = scalarBasis->localView();
   std::vector<Dune::FieldVector<double, 3>> stressVector(scalarBasis->size());
+  auto resultRequirements2 = Ikarus::ResultRequirements<>()
+                                 .insertGlobalSolution(Ikarus::FESolutions::displacement, displacement)
+                                 .addResultRequest(ResultType::PK2Stress);
+  std::vector<FiniteElement> fes;
+  fes.push_back(fe);
+  auto resultFunction = std::make_shared<ResultFunction<std::remove_cvref_t<FiniteElement>>>(&fes, resultRequirements2);
+  try {
+    vtkWriter.addPointData(Dune::Vtk::Function<GridView>(resultFunction));
```

Comment:
can you restrict this with constexpr std::is_same_v to the concrete Finite elements where this fails? Require that this fails for all elements is maybe a bit too much here. In general there should be a functor passed here, that does element specific things but this can be done in a seperate PR